ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v2/aemform/aemform.html

```diff
@@ -70,8 +70,10 @@
     <sly data-sly-test.formLocaleString="${form.locale || 'en'}"/>
         <iframe title="${form.title}"
                 role="${form.roleAttribute || ''}"
+                data-sly-set.dataRef="?${['dataRef=', request.requestParameterMap['dataRef'][0].toString] @ join=''}"
```

Comment:
Please add test cases for this use-case ensuring wcmmode disabled is not present in all the use-cases for both v1 and v2 forms